File: javascript/code_wars/keep_up_the_hoop.js
/**
 * The hoopCount function determines the level of proficiency in hula hooping.
 * It takes a number as an argument which represents the number of hoops a user can spin.
 *
 * @param {number} n - The number of hoops a user can spin.
 * @returns {string} - A motivational message based on the user's proficiency.
 * If the user can spin 10 or more hoops, the function returns "Great, now move on to tricks".
 * If the user can spin less than 10 hoops, the function returns "Keep at it until you get it".
 */
function hoopCount(n) {
    if (n >= 10) {
        return "Great, now move on to tricks";
    }
    return "Keep at it until you get it";
}

File: csharp/code_wars/keep_up_the_hoop.cs
/// <summary>
/// The Kata class.
/// </summary>
public class Kata
{
    /// <summary>
    /// Determines the encouragement message based on the number of hoops.
    /// </summary>
    /// <param name="n">The number of hoops.</param>
    /// <returns>A string message encouraging the user.</returns>
    public static string HoopCount(int n)
    {
        // If the number of hoops is 10 or more, return a message to move on to tricks
        if (n >= 10)
        {
            return "Great, now move on to tricks";
        }
        // If the number of hoops is less than 10, return a message to keep trying
        else
        {
            return "Keep at it until you get it";
        }
    }
}